client/packages/common/src/intl/context/IntlContext.tsx

```diff
@@ -39,7 +39,7 @@ export const IntlProvider: FC<PropsWithChildrenOnly> = ({ children }) => {
             },
             {
               /* options for secondary backend */
-              loadPath: './locales/{{lng}}/{{ns}}.json',
+              loadPath: '/locales/{{lng}}/{{ns}}.json',
```

Comment:
sorry my bad, i didn't check enough of the web app to see there is a break.
I had issues with this in electron package, i wanted to demonstrate that we can use translation string there, but perhaps it's not needed for now (that would be the simplest way). Otherwise I would need to look at exactly why locales are resolved to `file:///locales/...` vs `file:///{path to bundle}/locales/..`

Reply:
I've added to https://github.com/openmsupply/open-msupply/issues/58

Reply:
Thanks - I understand the issue now. This happens in the built version of the electron app:

<img width="822" alt="Screen Shot 2022-06-02 at 2 15 00 PM" src="https://user-images.githubusercontent.com/9192912/171540349-51689833-f7b3-4999-8c95-fa9a08f2c443.png">

To resolve, I've detected electron in 6292fc4c5